src/main/groovy/pl/allegro/tech/build/axion/release/ReleasePlugin.groovy

```diff
@@ -17,7 +17,7 @@ class ReleasePlugin implements Plugin<Project> {
     
     public static final String PUSH_RELEASE_TASK = 'pushRelease'
 
-    public static final String MARK_NEXT_VERSION_TASK = 'markNextVersionTask'
+    public static final String MARK_NEXT_VERSION_TASK = 'markNextVersion'
```

Comment:
this was a mistake, but thanks to Gradle guessing task names `markNextVersion` worked perfectly; since documentation holds `markNextVersion` and this is the only occurrence of `markNextVersionTask`, i assume changing it to the right value won't break any build